src/components/Loans/OpenLoans/components/OpenLoansSubHeader/OpenLoansSubHeader.js

```diff
@@ -111,6 +111,18 @@ class OpenLoansSubHeader extends React.Component {
     }));
   };
 
+  // Compare a particular status name (@targetStatus) to all the items in the checkedLoans object.
+  // Returns true if *all* the items have the target status, false otherwise.
+  onlyStatusSelected = (targetStatus) => {
```

Comment:
If we don't use this method anywhere, perhaps we should not declare it.

Reply:
👍 ⚡